integration_tests/requirements.txt

```diff
@@ -0,0 +1,6 @@
+pytest
+sre_yield
+pandas
+pyarrow
+pytest-xdist
```

Comment:
> we could add a follow up issue to add https://pypi.org/project/pytest-reqs or simply install these packages as part of run_pyspark _from_build

I prefer to let users install the requirements so that we don't automatically do something that may negatively affect their environment. I would want to make sure I have the correct virtual environment active before installing new packages, for example.

Reply:
I have added the minimum requirement of 2.0.0 for xdist